tests/testthat/_snaps/backend-sqlite.md

```diff
@@ -10,6 +10,36 @@
       ON (`LHS`.`x` IS `RHS`.`x`)
       
 
+# full and right join
+
+    Code
+      full_join(df1, df2, by = "x")
+    Output
+      <SQL>
+      SELECT `x`, `y.x`, `y.y`, `z`
+      FROM (SELECT COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`, `LHS`.`y` AS `y.x`, `RHS`.`y` AS `y.y`, `z`
+      FROM `df` AS `LHS`
```

Comment:
Hmmmm, this is a little confusing because `lazy_frame()` doesn't give you the ability to name the data frames. Not sure if it's worth doing anything about that.

Reply:
I fiddled around with swapping the names but this produces incorrect results. To avoid this one would need to allow custom table names in `sql_join_tbls()` and I also don't really think it is worth adding this extra complexity for basically one use case in one backend.

Reply:
I was thinking more about building some flexibility into `sql_build.op_base_local()` so that `lazy_frame()` could get an option name parameter. Either way, I don't think it's a blocker for this PR.